sdk/tests/features/unit/test_feature_file_writer.py

```diff
@@ -0,0 +1,31 @@
+import os
+import unittest
+
+from features.feature_extractor import extract_all_features_from_package
+from features.feature_file_writer import write_features_to_file
+from features.get_project_root import get_project_root
+
+
+def remove_file(filepath):
```

Comment:
do these need to be global functions? if they are only used in the `FeatureFileWriterTests` then I would move them into the class

Reply:
@rizwansaeed I put them all into the feature_file_writer module in case they will be used for any extra future tests